sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/highlighting/XmlHighlightingTest.java

```diff
@@ -269,6 +269,57 @@ public void tesCharBeforeProlog() throws Exception {
     assertData(highlightingData.get(4), 32, 39, "s");
   }
 
+  @Test
+  public void testBOM() throws Exception {
```

Comment:
@vilchik-elena It would be better to avoid duplications between this method and the next 2 methods.